### ISSUE TITLE
{{ events:between }} tag

### ISSUE THREAD

**Issue by enxoco** [transformstudios/statamic-events#17]
We are in the process of migrating a client site from Statamic v2 to v3.  We are using the events tag on this page in order to allow users to view/filter events based on dates, category, etc.

https://www.palmettodunes.com/calendar?from=2021-12-07&to=2021-12-15&q=&cat%5B0%5D=dining&page=1

Our template file looks something like this
```
      {{
        events:between
        collection="events"
        limit="5"
        offset="2"
        paginate="true"
        :taxonomy:event_categories="get:cat"
        :start="get:from"
        :end="get:to"
        :query="get:q"
        filter="calendar_search"
      }}
```

I did a little digging and found the between function in the old addon code base but not in the newer plugin codebase.  Is this a feature that was removed in the new version of the addon?  If so, is there similar functionality in the newer version?

**Comment by adnankussair** [comment]
Hey Mike. We're looking into this and will get back to you shortly. Stay tuned.

**Comment by enxoco** [comment]
@adnankussair Thanks.  This is what I have in ```EventsTags.php``` of the old version.
```
    public function between()
    {
        $now = Carbon::now();
        $nowPlusYear = Carbon::now()->addDays(366);
        $this->from = $this->getParam('from', $now);
        $this->to = $this->getParam('to', $nowPlusYear);

        $this->limit = $this->getInt('limit', 1);
        $this->offset = $this->getInt('offset', 0);

        $this->loadEvents($this->getBool('collapse_multi_days', false));

        $this->dates = $this->events->all($this->from, $this->to);

        if ($this->getBool('paginate')) {
            $this->paginateAll();
        }

        return $this->output();
    }
```

**Comment by edalzell** [comment]
Hey there @enxoco, thanks for pointing this out.

We'll get this added this month.

**Comment by enxoco** [comment]
@edalzell Hope you had a great holiday and maybe some time off!  Any updates on this issue?

**Comment by edalzell** [comment]
> @edalzell Hope you had a great holiday and maybe some time off! Any updates on this issue?

Looking at it now, planning to release a fix this week.

**Comment by edalzell** [comment]
Released in v3.2
